etc/ci/update-wpt-checkout

```diff
@@ -94,7 +94,7 @@ function unsafe_open_pull_request() {
     AUTH="${WPT_SYNC_USER}:${WPT_SYNC_TOKEN}"
     UPSTREAM="https://${AUTH}@github.com/${WPT_SYNC_USER}/servo.git"
     git remote add "${REMOTE_NAME}" "${UPSTREAM}" || return 2
-    git push -f "${REMOTE_NAME}" "${BRANCH_NAME}" || return 3
+    git push -f "${REMOTE_NAME}" "${BRANCH_NAME}" 2>&1 >/dev/null || return 3
```

Comment:
The redirections need to be swapped for this to work properly. See [the Bash hackers wiki](http://wiki.bash-hackers.org/howto/redirection_tutorial#order_of_redirection_ie_file_2_1_vs_2_1_file) or the [`etc/ci/upload_docs.sh`](https://github.com/servo/servo/blob/e6d9c251d1b9770aa520bf358b79d906a00d71bb/etc/ci/upload_docs.sh#L37-L39) file, where we have done the same thing.